### PR TITLE
Launch modals

### PR DESCRIPTION
**Pull request by ArtsiomYeliseyenka**

### CODE REVIEW

File: app/localization/translated/be.json [hunk 1]
```diff
@@ -119,6 +129,11 @@
   "MembersPage.permissionMap": "Карта правоў",
   "MembersPage.searchByName": "Шукаць па імя",
   "MembersPage.title": "Удзельнікі праекта",
+  "MembersPageToolbar.inviteUser": "Invite user",
+  "MembersPageToolbar.permissionMap": "Permission map",
+  "MembersPageToolbar.searchByName": "Search by name",
```
Comment (DiscoElevator) [inline app/localization/translated/be.json]: Possible duplicate with `MembersPage`?
I think `MembersPage` has been recently renamed to `MembersPageToolbar`. Please check the code for the right localization key.

File: app/src/components/main/grid/grid.jsx [hunk 2]
```diff
@@ -33,13 +42,18 @@ Grid.propTypes = {
...
   columns: [],
   data: [],
   sortingDirection: null,
   sortingColumn: null,
-  onChangeSorting: () => {
-  },
+  onChangeSorting: () => {},
   onFilterClick: () => {},
+  onDeleteItem: () => {},
+  onMoveToDebug: () => {},
```
Comment (DiscoElevator) [inline app/src/components/main/grid/grid.jsx]: Grid is a universal component. It shouldn't know anything about items it displays. 'Move to debug' operation is applicable only to launches, so it's not universal at all.

File: app/src/pages/inside/launchesPage/modals/launchDeleteModal/launchDeleteModal.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    data: {
+      item: {},
+      onConfirm: () => {},
+    },
+  };
+
+  render() {
+    const { intl } = this.props;
+    const { item, onConfirm } = this.props.data;
+    const confirmAndClose = (closeModal) => {
```
Comment (DiscoElevator) [inline app/src/pages/inside/launchesPage/modals/launchDeleteModal/launchDeleteModal.jsx]: Move to class method so it won't be created on every render

File: app/src/pages/inside/launchesPage/modals/launchEditModal/launchEditModal.jsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    intl: intlShape.isRequired,
+  };
+  componentDidMount() {
+    this.props.initialize(this.props.data.launch);
+  }
+  saveLaunchAndCloseModal = (closeModal) => (launch) => {
+    this.props.data.onEdit(launch);
+    closeModal();
+  };
+  formatToTagOptions = (tags) => tags.map((tag) => ({ value: tag, label: tag }));
```
Comment (DiscoElevator) [inline app/src/pages/inside/launchesPage/modals/launchEditModal/launchEditModal.jsx]: formatTags

File: app/src/pages/inside/launchesPage/modals/launchEditModal/launchEditModal.jsx [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+  formatToTagArray = (options) => options.map((option) => option.value);
```
Comment (DiscoElevator) [inline app/src/pages/inside/launchesPage/modals/launchEditModal/launchEditModal.jsx]: parseTags

File: app/src/pages/inside/launchesPage/modals/launchEditModal/launchEditModal.jsx [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+          <ModalField label={intl.formatMessage(messages.tags)}>
+            <FieldProvider
+              name="tags"
+              format={this.formatToTagOptions}
+              parse={this.formatToTagArray}
+            >
+              <FieldErrorHint>
+                <InputTagsSearch
+                  placeholder={intl.formatMessage(messages.tagsPlaceholder)}
+                  focusPlaceholder={'Please enter 1 or more characters'}
```
Comment (DiscoElevator) [inline app/src/pages/inside/launchesPage/modals/launchEditModal/launchEditModal.jsx]: localization

File: app/src/pages/inside/launchesPage/modals/moveToDebugModal/moveToDebugModal.jsx [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+  static defaultProps = {
+    data: {
+      onConfirm: () => {},
+    },
+  };
+
+  render() {
+    const { intl } = this.props;
+    const { onConfirm } = this.props.data;
+    const confirmAndClose = (closeModal) => {
```
Comment (DiscoElevator) [inline app/src/pages/inside/launchesPage/modals/moveToDebugModal/moveToDebugModal.jsx]: can be a class method

File: app/localization/translated/be.json (hunk 1 above)
Comment (ArtsiomYeliseyenka) [reply app/localization/translated/be.json]: fixed

File: app/src/pages/inside/launchesPage/modals/moveToDebugModal/moveToDebugModal.jsx (hunk 7 above)
Comment (ArtsiomYeliseyenka) [reply app/src/pages/inside/launchesPage/modals/moveToDebugModal/moveToDebugModal.jsx]: fixed

File: app/src/pages/inside/launchesPage/modals/launchEditModal/launchEditModal.jsx (hunk 6 above)
Comment (ArtsiomYeliseyenka) [reply app/src/pages/inside/launchesPage/modals/launchEditModal/launchEditModal.jsx]: fixed

File: app/src/pages/inside/launchesPage/modals/launchEditModal/launchEditModal.jsx (hunk 5 above)
Comment (ArtsiomYeliseyenka) [reply app/src/pages/inside/launchesPage/modals/launchEditModal/launchEditModal.jsx]: fixed

File: app/src/pages/inside/launchesPage/modals/launchEditModal/launchEditModal.jsx (hunk 4 above)
Comment (ArtsiomYeliseyenka) [reply app/src/pages/inside/launchesPage/modals/launchEditModal/launchEditModal.jsx]: fixed

File: app/src/pages/inside/launchesPage/modals/launchDeleteModal/launchDeleteModal.jsx (hunk 3 above)
Comment (ArtsiomYeliseyenka) [reply app/src/pages/inside/launchesPage/modals/launchDeleteModal/launchDeleteModal.jsx]: fixed

**Pull request merged**